### PR TITLE
fix(pagination): Align pagination select arrows

### LINKED ISSUE #764: Pagination select arrows not aligned correctly
## Detailed description

> Describe in detail the issue you're having.

The pagination component's select arrows are not aligned correctly with `carbon-components` >= `10.5.0`.

> Is this a feature request (new component, new icon), a bug, or a general issue?

It's a bug.

> Is this issue related to a specific component?

The pagination component.

> What did you expect to happen? What happened instead? What would you like to see changed?

I expected the select arrows to be aligned correctly like they are in the vanilla component. They're not aligned correctly. I would like to see the arrows be aligned correctly.

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

`10.6.2`, but the issue is present with versions >= `10.5.0`.

## Additional information
Pagination component

### PR DESCRIPTION
**Pull request by Donisius**
Closes #764 

The arrows are aligned in the storybook but when the pagination component is used outside of the storybook they are no longer aligned as shown below:

Screen Shot 2019-10-15 at 4 54 18 PM

The arrows are now aligned when the component is used both in and out of the storybook.

#### Changelog

**Changed**

* Changed the chevron-down-16 icon component to svg path in order to properly align the arrows.

### CODE REVIEW

File: src/pagination/pagination.component.ts [hunk 1]
```diff
@@ -129,12 +135,19 @@ export interface PaginationTranslations {
 						[(ngModel)]="currentPage">
 						<option *ngFor="let page of pageOptions; let i = index;" class="bx--select-option" [value]="i + 1">{{i + 1}}</option>
 					</select>
-					<ibm-icon-chevron-down16
+					<svg
```
Comment (cal-smith) [inline src/pagination/pagination.component.ts]: Can we use the directive form of the icon here? `<svg ibmIconChevronDown16></svg>`?

File: src/pagination/pagination.component.ts [hunk 2]
```diff
@@ -79,11 +79,17 @@ export interface PaginationTranslations {
 								{{ option }}
 						</option>
 					</select>
-					<ibm-icon-chevron-down16
-						style="display: inherit;"
-						innerClass="bx--select__arrow"
-						[ariaLabel]="optionsListText.subject | async">
-					</ibm-icon-chevron-down16>
+					<svg
```
Comment (cal-smith) [inline src/pagination/pagination.component.ts]: Same here?

File: src/pagination/pagination.component.ts (hunk 1 above)
Comment (Donisius) [reply src/pagination/pagination.component.ts]: The directive form worked! Made the changes.

File: src/pagination/pagination.component.ts [hunk 3]
```diff
@@ -80,15 +80,12 @@ export interface PaginationTranslations {
 						</option>
 					</select>
 					<svg
-						focusable="false"
-						preserveAspectRatio="xMidYMid meet"
-						style="will-change: transform;"
-						xmlns="http://www.w3.org/2000/svg"
+						[ngClass]="{'If this is here its the new chevron': true}"
```
Comment (cal-smith) [inline src/pagination/pagination.component.ts]: Probably didn't mean to leave these in here 😉

Comment (Donisius) [reply src/pagination/pagination.component.ts]: 🙃

**Pull request merged**